components/search_dropdown.tsx

```diff
@@ -2,118 +2,118 @@ import styles from "../styles/search_dropdown.module.css";
+    });
+    setSuggestions(newSuggestions);
+  };
 
-    return (
-        <div className={styles.searchContainer} ref={dropdownRef}>
-            <input
-                onChange={(e) => setSearch(e.target.value)}
-                value={search}
-                type="text"
-                className={styles.searchBar}
-                placeholder="Enter your Pincode or District name"
-                ref={inputRef}
-                onFocus={() => setShowSuggestions(true)}
-            />
-            {showSuggestions && (
-                <ul
-                    className={styles.suggestionsContainer}
-                    id="list-container"
-                    ref={listContainerRef}
-                    onClick={handleSelectSuggestionClick}
+  return (
+    <div className={styles.searchContainer} ref={dropdownRef}>
+      <input
+        onChange={(e) => setSearch(e.target.value)}
+        value={search}
+        type="text"
+        className={styles.searchBar}
+        placeholder="Enter your Pincode or District name"
+        ref={inputRef}
+        onFocus={() => setShowSuggestions(true)}
+      />
+      {showSuggestions && (
+        <ul
+          className={styles.suggestionsContainer}
+          id="list-container"
+          ref={listContainerRef}
+          onClick={handleSelectSuggestionClick}
+        >
+          {suggestions &&
+            suggestions.map((item, index) => {
+              return (
+                <li
+                  className={styles.suggestion}
+                  key={item[keyValue] + index}
+                  value={index}
                 >
-                    {suggestions &&
-                        suggestions.map((item, index) => {
-                            return (
-                                <li
-                                    className={styles.suggestion}
-                                    key={item[keyValue] + index}
-                                    value={index}
-                                >
-                                    {item[keyValue]}
-                                </li>
-                            );
-                        })}
-                    {suggestions.length === 0 && (
-                        <li
-                            className={`${styles.suggestion} ${styles.noSuggestion}`}
-                            key="no-suggestions"
-                        >
-                            No matching districts
-                        </li>
-                    )}
-                </ul>
-            )}
-        </div>
-    );
+                  {item[keyValue]}
+                </li>
+              );
+            })}
+          {suggestions.length === 0 && (
+            <li
+              className={`${styles.suggestion} ${styles.noSuggestion}`}
+              key="no-suggestions"
+            >
+              No matching districts
+            </li>
+          )}
+        </ul>
+      )}
+    </div>
+  );
```

Comment:
Just wanted to confirm is there any specific changes or it is just lint fix.

Reply:
@chavda-bhavik Could you run the following command on the PR

```
npm run prettify
```

Reply:
Yes, @kshitizshakya, it was formatted while running `npm run prettify`, and @preetjdp `npm run prettify` isn't making any changes, as I already run it before committing.